src/event_parser/event_parser.y

```diff
@@ -228,7 +223,8 @@ opt:
                delete $3;
                VAL_ERROR();
              }
-             g_accum_event_conf.mode |= *mode;
+             // override mode if present
+             g_accum_event_conf.mode = *mode;
```

Comment:
I don't think we should allow several modes (though we don't check that afaik...)